src/drivers/mct/cime_config/config_component_cesm.xml

```diff
@@ -52,7 +52,7 @@
   <entry id="INFO_TASKMAP_MODEL">
     <type>integer</type>
     <valid_values>0,1,2</valid_values>
-    <default_value>0</default_value>
+    <default_value>1</default_value>
```

Comment:
Is there really a need for another variable here?  Why not just base this on info_mprof > 2?  That seems to be the requirement below.

Reply:
`*TASKMAP*` and `*MPROF*` serve different uses: first one is for checking task-to-node allocations, second one is for profiling memory growth and out-of-memory errors in high-resolution cases. Per-task memory profiling (info_mprof={1,2}) provides relative and finer granularity for tracking component ROOTPE and PIO_ROOT vs. other tasks. Task-to-node mapping can be reimplemented inside mem-prof to make the two independent.